osi_object.proto

```diff
@@ -436,6 +437,84 @@ message MovingObject
         // through.
         //
         optional double ground_clearance = 6;
+
+        // Description of each wheel.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated WheelData wheel_data = 7;
+
+        // \brief Detailed wheel data.
+        // The focus is on the description of a wheel regarding the perceivable 
+        // information from the outside. 
+        // It is not intended to be used for e.g. dynamic calculations.
+        //
+        message WheelData
+        {
+            // The axle which contains this wheel. A value of 0 represents the
+            // front-most axle of the vehicle with higher numbers incrementing
+            // towards the rear-most axle.
+            //
+            optional uint32 axle = 1;
+
+            // The index of the wheel on the axle, counting in the direction
+            // of positive-y, i.e. right-to-left.
+            //
+            // For example, on a standard 2-axle, 4-wheel car, the rear-right
+            // wheel would be (axle=1, index=0).
+            // This concept works also for twin tires.
+            //
+            optional uint32 index = 2;
+            
+            // Position of the wheel (geometric center of the wheel).
+            // The reference system is the vehicle frame (center of bounding box).
+            // Right-handed, x-axis shows into driving direction "forward".
+            // Rough Approximation: The z-value shows the damping.
+            //
+            optional Vector3d position = 3;
+
+            // Median radius of the wheel measured from the center of the wheel to
+            // the outer part of the tire.
+            //
+            // Unit: m
+            //
+            optional double wheel_radius = 4;
+
+            // Median radius of the rim measured from the center to the outer, visible part of the rim.
+            //
+            // Unit: m
+            //
+            optional double rim_radius = 5;
+            
+            // Median width of the tire.
+            //
+            // Unit: m
+            //
+            optional double width = 6;
+
+            // Orientation of the wheel.
+            //
+            // The reference system is the vehicle frame (center of bounding box).
+            // Right-handed relative to the vehicle's forward motion.  The yaw-value shows the steering angle.
+            //
+            optional Orientation3d orientation = 7;
+
+            // Rotation rate in the wheel's frame of reference (rotation around the wheel axle).
```

Comment:
This is a bit tricky for me to understand. Did I got it right: 
There is a wheel frame that you use as a reference. The wheel frame used in the simulation may vary in size. You "measure" the distance of the wheel frame used in the simulation for one complete rotation and relate this to the reference wheel frame. That means the value describes how often the reference wheel frame rotates around its axle to equal the distance that the wheel frame used in the simulation would cover in one complete rotation.

Reply:
I am afraid I don't really get you point, let's discuss this offline.

Reply:
I simplified that, please have a check @paagkame and @kmeids